src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        if(numbers.size() != 6) {
+            throw new IllegalArgumentException("로또번호는 6개여야 합니다.");
+        }
+        this.numbers = numbers;
+    }
+
+    public List<Integer> getNumbers() {
+        return this.numbers;
+    }
+
+    public int countMatches(List<Integer> winningNumbers) {
```

Comment:
```suggestion
    public int countMatches(Lotto winningNumbers) {
```
winningNumbers도 6개의 숫자를 다루고 있어 Lotto 객체로 봐도 되지 않을까요?